modules/nuxeo-apidoc-webengine/src/main/java/org/nuxeo/apidoc/browse/BundleGroupWO.java

```diff
@@ -37,6 +40,15 @@ public NuxeoArtifact getNxArtifact() {
         return getTargetBundleGroup();
     }
 
+    @Produces("text/html")
+    @Override
+    public Object doViewDefault() {
+        Template t = (Template) super.doViewDefault();
+        BundleGroup group = getTargetBundleGroup();
```

Comment:
nit: would probably drop this local var and just chain `getTargetBundleGroup().getReadmes()...`